www/_posts/2020-06-01-cordova-ios-release-6.0.0.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova iOS 6.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 6.0.0`! This is Cordova's official platform for building iOS mobile applications.
+
+* [cordova-ios@6.0.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
+
+
+**To upgrade:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@6.0.0
+```
+
+The most notable changes in this major release are:
+
+* [GH-790](https://github.com/apache/cordova-ios/pull/790) breaking feature: Integrate & replace SplashScreens w/ Launch Storyboard (CB-13143)
+* [GH-849](https://github.com/apache/cordova-ios/pull/849) breaking(`pod`): remove unused API & bump minimum version requirements to `1.8.0`
+* [GH-820](https://github.com/apache/cordova-ios/pull/820) Only set `bundleid` to main app target
+* [GH-801](https://github.com/apache/cordova-ios/pull/801) breaking (`UserAgent`): Drop `CDVUserAgentUtil` and Implement for WKWebView
+* [GH-781](https://github.com/apache/cordova-ios/pull/781) breaking: Use `WKURLSchemeHandler` for serving app content
+* [GH-780](https://github.com/apache/cordova-ios/pull/780) breaking: upgrade xcode compatible to 11.0
+* [GH-779](https://github.com/apache/cordova-ios/pull/779) breaking: bump `deployment-target` to `11.0`
+* [GH-773](https://github.com/apache/cordova-ios/pull/773) refactor: drop `uiwebview` & add `wkwebview`
+
+We have removed the `UIWebView` webview from our core platform and integrated `WKWebView` as its default. This change makes the `cordova-plugin-wkwebview-engine` plugin obsolete.
+
+To resolve some of the known `WKWebView` issues, such as CORS, the `WKURLSchemeHandler` has been implemented. The `WKURLSchemeHandler` is necessary to serve your app content on custom scheme. By default, Cordova iOS will continue to server on the `file` scheme. You are able to change the scheme by setting the preference options `scheme` and `hostname` in the `config.xml` file.
+
+E.g.
+
+```xml
+<preference name="scheme" value="app" />
+<preference name="hostname" value="localhost" />
+```
+
+In order to support `WKURLSchemeHandler`, the `deployment-target` was bumped to `11.0`.
```

Comment:
This should more clearly call out that this changes the minimum supported iOS version and drops support for iOS 10.